.changeset/beige-rockets-notice.md

```diff
@@ -0,0 +1,18 @@
+---
+'@livepeer/react': minor
+---
+
+**Feature:** Asset Metrics (`useAssetMetrics`)
+
+`useAssetMetrics` hook has been added for fetching asset metrics. This hook will update
+when viewership metrics have been reported to the correct reporting URL (this is handled in
+`@livepeer/react`'s `VideoPlayer`).
+
+```typescript
+const { data: metrics } = useAssetMetrics({
+  assetId: createdAsset?.id,
+  refetchInterval: (metrics) => (!metrics ? 30000 : false),
```

Comment:
Maybe this should be the default inside `useAssetMetrics` unless specified otherwise?

Reply:
I think we should leave this up to the consumer and specify in the docs. I don't think forcing a refetch interval would be an expected pattern

Reply:
Hmm I see. I thought that's what we did on the `useAsset/Stream`s already! If not, then I agree that it doesn't make sense to have it only here.

Reply:
Hmm, thinking further about this, was this logic only here while the viewership API was unstable, so you needed to refetch until some value was returned? If so, maybe we should simplify these examples and either only suggest a fixed refetch interval or no custom config at all (or some other config, maybe `refetchOnWindowFocus`?)

Reply:
These docs are just demonstrating what's possible with react-query under the hood, not what's required

Reply:
Since this is a changeset

Reply:
Ah ok! Lets leave it as is then